source/detail/serialization/xlsx_consumer.cpp

```diff
@@ -158,10 +164,10 @@ cell xlsx_consumer::read_cell()
 
     auto ws = worksheet(current_worksheet_);
 
-    if (in_element(qn("spreadsheetml", "sheetData")))
+    if (in_element(sheetData_el))
     {
-        expect_start_element(qn("spreadsheetml", "row"), xml::content::complex); // CT_Row
-        auto row_index = parser().attribute<row_t>("r");
+        expect_start_element(row_el, xml::content::complex); // CT_Row
+        auto row_index = static_cast<row_t>(std::stoul(parser().attribute("r")));
```

Comment:
For whatever reason, this is dramatically faster than how `libstudxml` does the conversion